tests/runtime/regression

```diff
@@ -52,5 +52,20 @@ TIMEOUT 1
 
 NAME strncmp_checks_for_nul
 RUN {{BPFTRACE}} -e 'BEGIN { printf("_%d_\n", strncmp("one", "one ", 4)); exit() }'
-EXPECT _0_
+EXPECT _1_
```

Comment:
I believe that the expected result was incorrect as the two strings are not equal. `strncmp` in C also returns a non-zero result in this case.

Reply:
Oops, sorry